app/controllers/pieces_controller.rb

```diff
@@ -10,9 +10,9 @@ def create
 
   def update
     @piece = Piece.find(params[:id])
-    @piece.move_to!(:x_position, :y_position) 
-    @game.update(white_player_turn: !@game.white_player_turn)
-    redirect_to game_path(@game)
+    @piece.move_to!(params[:x_position], params[:y_position])
+    redirect_to game_path(current_game[:id])
+    @current_game.update(white_player_turn: !@current_game.white_player_turn)
```

Comment:
Nice job @PaleoGuy2012 

I'd suggest to move the whole logic into the `move_to` method and add a method on the game model.. sth like `switch_turns!` which do whatever the line 15 does.. and in the `move_to` of the piece you can just say: `game.switch_turns!` hope it makes sense

Reply:
That does, I'll work on that good suggestion.  Keep those controllers skinny.